packages/cli/src/commands/generate/service/__tests__/service.test.js

```diff
@@ -1,146 +1,230 @@
 global.__dirname = __dirname
+
 import { loadGeneratorFixture } from 'src/lib/test'
+import { getDefaultArgs } from 'src/lib'
 
 import * as service from '../service'
 
-test('returns exactly 2 files', async () => {
-  const files = await service.files({
-    name: 'User',
-    crud: false,
-    relations: [],
-  })
+const extensionForBaseArgs = (baseArgs) =>
+  baseArgs && baseArgs.typescript ? 'ts' : 'js'
 
-  expect(Object.keys(files).length).toEqual(2)
-})
+const itReturnsExactly2Files = (baseArgs) => {
```

Comment:
noted in a comment elsewhere - extracted tests into named function to reduce duplication across TS and JS versions